cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
This would break Container and Middleware... I'm not sure this behavior should be defined in `BaseProvider`. I'd rather see it specifically under `cloud.Provider`, leaving the other providers as they are.